prow/cmd/sinker/main.go

```diff
@@ -130,15 +133,20 @@ func main() {
 	}
 
 	// Clean now and regularly from now on.
-	for {
+	start := time.Now()
+	c.clean()
+	logrus.Infof("Sync time: %v", time.Since(start))
+	if o.runOnce {
+		os.Exit(0)
```

Comment:
Is this intentionally avoiding graceful shutdown?  If not we should be using `return` so that the deferred `WaitForGracefulShutdown()` still runs.

Reply:
Yeah it's changed locally